content/docs/upgrading/upgrade.md

```diff
@@ -4,11 +4,67 @@ description = "Upgrading your deployment to a later version of Kubeflow"
 weight = 30
 +++
 
-Until version 1.0, Kubeflow makes no promises of backwards compatibility or 
-upgradeability. There is **no clean upgrade path** to the latest version of 
-Kubeflow ({{% kf-latest-version %}}).
+Upgrading your Kubeflow deployment is supported if your deployment is v0.7.0 or later.
 
-Nonetheless, here are some instructions for updating your deployment:
+Prerequisites:
+
+* Download the latest kfctl binary from the
+  [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/tag/{{% kf-latest-version %}}).
+
+* Ensure that your kubeflow namespace is annotated with the following label:
+```
+  labels:
+    control-plane: kubeflow
+```
+* You must have a local Kubeflow application directory matching your current
+   deployment.
+
+Upgrade instructions:
+
+1. Create an upgrade spec in the parent directory of the Kubeflow application. An example is provided
```

Comment:
s/spec/specification

It's best not to use abbreviations, as they can be difficult for people to understand and can lead to confusion when translating to other languages.